libfreerdp/codec/interleaved.c

```diff
@@ -158,9 +160,9 @@ static INLINE BOOL buffer_within_range_(const void* pbSrc, const void* pbEnd, co
                                         const char* file, size_t line)
 {
 	WINPR_UNUSED(file);
-	if (pbSrc >= pbEnd)
+	if (pbSrc > pbEnd)
 	{
```

Comment:
I think the previous version was the correct one.

Reply:
no.